### DISCUSSION TITLE
What is a hidden button and where is it located?

### DISCUSSION THREAD

**Discussion by ghost** [Authenticator-Extension/Authenticator#1166] [General]
The translation here mentions a hidden button, but I could not find this button on the application. There is only a pin icon to the right of the accounts.
Can you please provide a screenshot showing where this button is located?

![hidden](https://github.com/Authenticator-Extension/Authenticator/assets/153795294/74bc284e-e6c4-442e-9992-3338aefff993)

**Comment by Sneezry** [comment]
It's the QR button next to the pin button. The QR button is hidden before you hover your cursor over the entry. Clicking on this button will open a QR code for the account, allowing you to add it to other 2FA clients.

**Comment by ghost** [comment]
I understand, thanks.
